frontend-react/src/App.tsx

```diff
@@ -1,88 +1,83 @@
-        axios
-            .post(
-                "/plan-it/user/token",
-                {
-                    code: response.code,
-                    planit_userId: planitUserId
-                },
-                {
-                    headers: {
-                        "Bypass-Tunnel-Reminder": 1,
-                    },
-                }
-            )
-            .then((response) => {
-                console.log(response);
-                //here add set for local storage
-                setPlanitUserId(response.data.planit_userId);
-                console.log(planitUserId);
-            })
-            .catch((error) => console.log(error.message));
-    },[planitUserId]);
+function App() {
+  const [owner, setOwner] = useState('');
+  const [accountEmails, setAccountEmails] = useState<string[]>([]);
+  const [planitUserId, setPlanitUserId] = useLocalStorage('planitUserId', null);
 
-    const onEventSubmit = (result: EventCreateRequest) => {
-        console.log("succ: ", result);
-        axios
-            .post("/plan-it/calendar/new-event", result)
-            .then((response) => {
-                console.log(response);
-            })
-            .catch((error) => console.log(error.message));
-    };
+  const onSuccess = useCallback(
+    (response: any) => {
+      console.log('succ: ', response);
+      axios
+        .post(
+          '/plan-it/user/token',
+          {
+            code: response.code,
+            planit_userId: planitUserId,
+          },
+          {
+            headers: {
+              'Bypass-Tunnel-Reminder': 1,
+            },
+          }
+        )
+        .then((response) => {
+          console.log(response);
+          //here add set for local storage
+          setPlanitUserId(response.data.planit_userId);
+          console.log(planitUserId);
+        })
+        .catch((error) => console.log(error.message));
+    },
+    [planitUserId, setPlanitUserId]
+  );
 
-    const getEmailList = (id:string) => {
-        axios
-            .get(`/plan-it/user/getAllEmails/${id}`)
-            .then(response => setAccountEmails(response.data))
-    }
+  const onEventSubmit = (result: EventCreateRequest) => {
+    console.log('succ: ', result);
+    axios
+      .post('/plan-it/calendar/new-event', result)
+      .then((response) => {
+        console.log(response);
+      })
+      .catch((error) => console.log(error.message));
+  };
 
-    useEffect(() => {
-        if (planitUserId !== null)
-            getEmailList(planitUserId);
-    }, [planitUserId])
+  const getEmailList = (id: string) => {
+    axios.get(`/plan-it/user/getAllEmails/${id}`).then((response) => setAccountEmails(response.data));
+  };
 
-    // @ts-ignore
-    const login = useGoogleLogin({
-        onSuccess: onSuccess,
-        flow: "auth-code",
-        scope:
-            "profile email openid https://www.googleapis.com/auth/calendar https://www.googleapis.com/auth/calendar.readonly",
-        accessType: "offline",
```

Comment:
Why get rid of accessType: "offline"?

Reply:
When removed `@ts-ignore`, I found out that TypeScript didn't like `accessType` field being set there. After a bit of research it turned out that there isn't such a parameter in the documentation at all: https://www.npmjs.com/package/@react-oauth/google

Therefore I removed it, and after successfully testing the auth flow, I decided it's redundant and it's safe to remove.

Reply:
i guess it was a hold over from the previous library to interact with google

Reply:
yeah